applications/Unity.GrantManager/src/Unity.GrantManager.Web/Views/Shared/Components/AssessmentResults/Default.js

```diff
@@ -85,7 +85,30 @@
     initDatePicker();
 });
 
-
+let dueDateHasChanged = false;
```

Comment:
@aurelio-aot - will this not work?

The save becomes this:
function enableResultSaveBtn(element) {    
    if ($("#assessmentResultForm").valid()) {
        $('#saveAssessmentResultBtn').prop('disabled', false);
    }
    else {
        $('#saveAssessmentResultBtn').prop('disabled', true);
    }
}

Not sure i see the need for the individual validation and variables?

Reply:
hi @AndreGAot -- if the Due Date was saved previously and the Due Date is past date now, the code above will prevent the form from getting saved.  In my example below,  I only changed the Likelihood of Funding and not the Due Date but the Due Date is getting validated and preventing the form to be submitted.

![image](https://github.com/bcgov/Unity/assets/97022120/62ad0ffd-442f-4c5e-8a3e-24a8e0fa1402)

Reply:
okay, i see - catering for legacy data

Reply:
Let try refactor this and highlight that it is for legacy data captured, and also do validations in the API